helix-term/src/commands.rs

```diff
@@ -2267,6 +2268,17 @@ fn file_picker(cx: &mut Context) {
     cx.push_layer(Box::new(overlayed(picker)));
 }
 
+fn file_picker_in_current_view_directory(cx: &mut Context) {
```

Comment:
We should call this `file_picker_in_current_buffer_directory` like #3063. We could call in "document" instead to match the internals but in other commands like `:buffer-close` we already call documents buffers